_posts/2025-09-17-cockpit-348.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Cockpit 347
+author: lis
+date: '2025-09-17'
+tags: cockpit, machines
+slug: cockpit-347
+category: release
+summary: ''
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 347 and cockpit-machines 340:
+
+
+## Site-specific branding support
+
+Administrators can provide a custom local [branding](https://github.com/cockpit-project/cockpit/blob/main/doc/branding.md) in `/etc/cockpit/branding/` (more generally, `$XDG_CONFIG_DIRS/cockpit/branding/`). If that directory exists, it is considered before the system package branding directories in /usr.
+
+## Machines: Exclusive VNC connections with "Remote resizing"
+
+A VNC viewer can tell the VNC server whether or not it wants exclusive access to it, or whether it is okay that multiple viewers display and interact with the same console. Until now, Cockpit has always allowed shared access when it connects.
+
+Now, Cockpit will request exclusive access when it is in the "Remote resizing" mode, to avoid having multiple viewers make conflicting resize requests.
+
+The feature requires the web server from Cockpit 346.
```

Comment:
perhaps "The" → "This" or "That"? Can't tell you exactly why, but "The" sounds weird in my ears.